cfpb/cfgov-refresh
# Add self-hosted fonts option

Adds self-hosted font update from CF.

## Removals

- `static-legacy/css/licensed-fonts.css`

## Changes

- Replaces `gulp-mq-remove` with `postcss-unmq`.
- Replaces `gulp-autoprefixer` with `autoprefixer`.
- Brings in latest CF updates.

## Testing

1. `./frontend.sh production` should pass.
2. Vising the site locally should be pulling the fonts from fonts.net.
3. change `@use-font-cdn: true;` to `false` inside `main.less` and re-build the assets with `gulp clean && gulp build` and the fonts should be trying to pull from localhost.
4. Check in an IE8 VM to ensure the page isn't broken looking.

## Notes

- `gulp-mq-remove` has a dependency `css-parse` that was failing on the new conditional code inside cf-typography for the font-face rule. Both of those modules haven't been updated in years, so I went with replacing them with a postcss option.

## Todos

- Still more work needs to be done in ansible on the backend to switch `@use-font-cdn: true;` to false.
